src/test/java/chess/dao/PieceDaoTest.java

```diff
@@ -59,44 +67,34 @@ void end() {
         jdbcTemplate.execute("DROP TABLE game IF EXISTS");
     }
 
+    @DisplayName("createAllByGameId 실행시 해당 gameId에 piece 정보들이 추가된다.")
     @Test
-    void createAllById_Piece_생성_성공() {
-        gameDao.createById("1234");
-
-        final ChessmenInitializer chessmenInitializer = new ChessmenInitializer();
-        final List<Piece> pieces = chessmenInitializer.init().getPieces();
-        pieceDao.createAllById(pieces, "1234");
+    void createAllById() {
+        pieceDao.createAllByGameId(pieces, gameId);
 
-        assertThat(pieceDao.findAllByGameId("1234").getPieces().size()).isEqualTo(32);
+        assertThat(pieceDao.findAllByGameId(gameId).getPieces().size()).isEqualTo(32);
     }
 
+    @DisplayName("updateAllByGameId 실행시 해당 gameId의 piece들의 정보가 바뀐다.")
     @Test
-    void updateAllByGameId_Piece_업데이트_성공() {
-        gameDao.createById("1234");
-        final ChessmenInitializer chessmenInitializer = new ChessmenInitializer();
-        final List<Piece> pieces = chessmenInitializer.init().getPieces();
-        pieceDao.createAllById(pieces, "1234");
-
-        pieces.remove(pieces.size()-1);
+    void updateAllByGameId() {
+        pieceDao.createAllByGameId(pieces, gameId);
+        pieces.remove(pieces.size() - 1);
         pieces.add(new King(Color.BLACK, Position.of("h2")));
 
-        pieceDao.updateAllByGameId(pieces, "1234");
-        Pieces move = pieceDao.findAllByGameId("1234");
-
+        pieceDao.updateAllByGameId(pieces, gameId);
+        Pieces moved = pieceDao.findAllByGameId(gameId);
 
-        assertThat(move.extractPiece(Position.of("h2")).getName()).isEqualTo("king");
+        assertThat(moved.extractPiece(Position.of("h2")).getName()).isEqualTo("king");
     }
 
+    @DisplayName("updateAllByGameId 실행시 해당 gameId의 piece들이 사라진다.")
     @Test
-    void deleteAllByGameId_Piece_삭제_성공() {
-        gameDao.createById("1234");
-
-        final ChessmenInitializer chessmenInitializer = new ChessmenInitializer();
-        final List<Piece> pieces = chessmenInitializer.init().getPieces();
-        pieceDao.createAllById(pieces, "1234");
-        pieceDao.deleteAllByGameId("1234");
+    void deleteAllByGameId() {
+        pieceDao.createAllByGameId(pieces, gameId);
+        pieceDao.deleteAllByGameId(gameId);
 
-        assertThat(pieceDao.findAllByGameId("1234").getPieces().size()).isEqualTo(0);
+        assertThat(pieceDao.findAllByGameId(gameId).getPieces().size()).isEqualTo(0);
```

Comment:
```suggestion
        assertThat(pieceDao.findAllByGameId(gameId).getPieces()).hasSize(0);
```

`AssertionsForClassTypes`가 아닌 `Assertions`의 `assertThat()`을 사용하면, 이런 편의 메서드들을 제대로 사용할 수 있어요.

Reply:
Assertion에도 다양한 import가 존재하는군요.! hasSize()뿐만 아니라 더 많은 메서드가 있는 것도 확인했습니다. 감사합니다 :)